### PR TITLE
Gradle Task 2

### PR DESCRIPTION
**Pull request by explosion204**

### CODE REVIEW

File: api/src/main/java/com/karnyshov/api/App.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.karnyshov.api;
+
+import com.karnyshov.core.Utils;
+
+public class App {
+    public static void main(String[] args) {
+        System.out.println(Utils.isAllPositiveNumbers("12", "79"));
```
Comment (NatalliaKhadunai) [inline api/src/main/java/com/karnyshov/api/App.java:7]: We should never ever use `System.out.println` in our code. But let's leave it as-is for now.
Just remember not to use it anywhere else.

File: core/src/main/java/com/karnyshov/core/Utils.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.karnyshov.core;
+
+import com.karnyshov.util.StringUtils;
+
+import java.util.Arrays;
+
+public class Utils {
```
Comment (NatalliaKhadunai) [inline core/src/main/java/com/karnyshov/core/Utils.java:7]: In another review, you've created a private constructor for the utils class. Please do the same thing here as well.

**Review by NatalliaKhadunai — CHANGES_REQUESTED** [review]
Left one comment that we need to fix.

File: core/src/main/java/com/karnyshov/core/Utils.java (hunk 2 above)
Comment (explosion204) [reply core/src/main/java/com/karnyshov/core/Utils.java:7]: fixed

**Comment by NatalliaKhadunai** [comment]
Well done!!

**Pull request merged**